eng/Packaging.props

```diff
@@ -24,11 +24,14 @@
     <!-- We're currently not building a "live" baseline, instead we're using .NETCore 1.0 RTM stable versions as the baseline -->
     <SkipBaseLineCheck>true</SkipBaseLineCheck>
 
+    <!-- The .NET Core product branding version -->
```

Comment:
This belongs in https://github.com/dotnet/corefx/blob/master/eng/Versions.props.  You can consider putting InformationalVersion there too if the evaluation order works out.  if not it's ok to stay in versioning.targets.

Reply:
Looks like you do have an evaluation order problem.  https://github.com/dotnet/corefx/blob/9fa041a04df9a38ab4167e76fbee6133eb627f71/Directory.Build.props#L228-L253

Packaging.props would need to come after sdk.props (which eventually imports eng\version.props).

This should be OK, but you can test it using the following methodology:
1. Add some empty target eg: <Target Name="Empty" />
2. Build before your change, dotnet msbuild /t:Empty /flp:v=diag;logfile=before.log
3. Build after your change, dotnet msbuild /t:Empty /flp:v=diag;logfile=after.log

Diff the diag logs to see if any property values changed (other than the one you're intending).

Reply:
@ericstj Thanks for the hint. Verified that there are no changes.